src/components/Goods.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { AppProps, prodData, propsGods } from '../globalTypes';
+import Product from './Product';
+
+const Goods = (props: propsGods) => {
+
+    const createGoods = () => {
```

Comment:
There is no need to create a function. But if it is, maybe it's better to move it to another module.